examples/pytorch/schnet/README.md

```diff
@@ -0,0 +1,36 @@
+SchNet & MGCN
+============
+- K.T. Schütt. P.-J. Kindermans, H. E. Sauceda, S. Chmiela, A. Tkatchenko, K.-R. Müller.
+SchNet: A continuous-filter convolutional neural network for modeling quantum interactions. Advances in Neural Information Processing Systems 30, pp. 992-1002 (2017) [link](http://papers.nips.cc/paper/6700-schnet-a-continuous-filter-convolutional-neural-network-for-modeling-quantum-interactions)
+- C. Lu, Q. Liu, C. Wang, Z. Huang, P. Lin, L. He, Molecular Property Prediction: A Multilevel Quantum Interactions Modeling Perspective. The 33rd AAAI Conference on Artificial Intelligence (2019) [link](https://arxiv.org/abs/1906.11081)
+
+Dependencies
+------------
+- PyTorch 1.0+
+- dgl 0.3+
+- RDKit (If use Alchemy dataset.)
+
+Usage  
+-----
+
+Example usage on Alchemy dataset:
+
++ SchNet: excepted MAE 0.065
```

Comment:
Could we reproduce this result with your code? How long would it take to train the model for an epoch?

Reply:
Yes, you can reproduce it with the default setting.
SchNet takes about 85 seconds per epochs.
MGCN takes about 110 seconds per epochs.

Reply:
Thank your for letting me know.